README.md

```diff
@@ -13,23 +13,43 @@ We provide a free, hosted platform that makes it easy to connect apps and develo
 **Key Features**:
 
```

Comment:
Mainly refactored this to reference component actions. Also modified some old links / language.